test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
> I'm not sure if it would actually cause problems

Anyway, if it's not a problem, it would probably be convenient to be in XRP
scale, given drops aren't really a thing in this api.